apps/src/codebridge/FileBrowser/FileBrowserRow/hooks/useFileRowOptions.ts

```diff
@@ -92,20 +100,31 @@ export const useFileRowOptions = (
         labelText: codebridgeI18n.deleteFile(),
         clickHandler: () => openConfirmDeleteFile({file}),
       },
-    ],
-    [
-      appName,
-      editableFileTypes,
-      file,
-      isLocked,
-      isStartMode,
-      openConfirmDeleteFile,
-      openMoveFilePrompt,
-      openRenameFilePrompt,
-      projectFiles,
-      projectFolders,
-    ]
-  );
+    ];
+
+    if (experiments.isEnabled(experiments.PYTHONLAB_BACKPACK)) {
+      options.push({
+        condition: true,
+        iconName: 'backpack',
+        labelText: 'Save to backpack',
```

Comment:
follow up item: we should translate this string and the other hard-coded english strings

Reply:
Oh yes! I meant to add that in the follow-up section. Will update section to include.